src/transport_ofi.c

```diff
@@ -1432,6 +1443,12 @@ int shmem_transport_init(void)
         shmem_transport_ofi_stx_allocator = ROUNDROBIN;
     }
 
+    if (shmem_internal_params.OFI_STX_DISABLE_PRIVATE) {
+        shmem_transport_ofi_disable_private = 1;
+    } else {
+        shmem_transport_ofi_disable_private = 0;
+    }
```

Comment:
Unless the OFI transport layer overrides the param set by the user, it's preferred to use the param directly rather than duplicating it into `shmem_transport_ofi_disable_private`.  Can you just check `shmem_internal_params.OFI_STX_DISABLE_PRIVATE` within `shmem_transport_ofi_is_private` and remove the second arg to this function?

Reply:
👍